precise_driver/src/precise_hw_interface.cpp

```diff
@@ -229,6 +231,8 @@ namespace precise_driver
         resetController(false);
         res.success = device_->releasePlate(req.width*1000, req.speed);
         res.success = device_->waitForEom();
+        //hack
+        res.success = true;
```

Comment:
what about this hacks?
any better solution?

Reply:
I don't think so for now. The service is functioning correctly. I would keep it/leave it as it is.